internal/npm_install/npm_install.bzl

```diff
@@ -303,6 +312,18 @@ npm_install = repository_rule(
 See npm CLI docs https://docs.npmjs.com/cli/install.html for complete list of supported arguments.""",
             default = [],
         ),
+        "npm_ci": attr.bool(
```

Comment:
a boolean is a strange way to model a string.

what should a user expect that `npm_ci = False` means? I don't think it would be very clear that it means "use npm install instead"

maybe it should be

`"npm_command": attr.string(default = "ci", doc = "which npm command to run to install dependencies", values = ["ci", "install"])`

Reply:
A very good idea. I was not 100% happy with the naming but I don't have something better in mind. Your suggestion is a perfect solution, will adapt the changes :)